### PR TITLE
#FN-39 feat: view and update user profile

### PR DESCRIPTION
**Pull request by EvelyneUmubyeyi**
# PR Description
In this PR I created pages for viewing and updating user profiles.

## Description of tasks that were expected to be completed

List all the tasks and sub-tasks that were assigned to you Make sure you check every completed task. list them in this style

- [ ] Create a view profile page
- [ ] Create update profile page

## Description of tasks that were expected to be completed and that are not yet completed

- [ ] None

## How this may be tested?

Please describe the tests that you ran to verify your functionalities. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

1. run `yarn i` to install all required dependencies
2. run `yarn run dev`
3. Go to profile page /profile and update profile page /updateProfile 

## Screenshots (If appropriate/applicable)

Provide screenshots related to your PR in case it's appropriate and applicable

## Please check this Checklist before you submit your PR:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] My code generates no warnings
- [ ] There are no vulnerabilities

### CODE REVIEW

**Review by Charlesntwari — APPROVED** [review]

**Review by Charlesntwari — APPROVED** [review]

File: vite.config.ts
Comment (Honore123) [inline vite.config.ts:1]: Add environmentPlugin so that we can use process in util api.tsx

File: src/utils/api.tsx [hunk 1]
```diff
@@ -1,8 +1,10 @@
 import axios from 'axios';
+import dotenv from 'dotenv';
 
+dotenv.config()
```
Comment (Illustre13) [inline src/utils/api.tsx]: Remove the dotenv import here

**Review by Illustre13 — CHANGES_REQUESTED** [review]

File: src/utils/api.tsx (hunk 1 above)
Comment (Illustre13) [reply src/utils/api.tsx]: Okay

**Review by Illustre13 — APPROVED** [review]

**Review by Honore123 — APPROVED** [review]

**Review by dushimeemma — CHANGES_REQUESTED** [review]
Please show the profile and edit profile pages if and if the user is authenticated.

**Review by dushimeemma — APPROVED** [review]

**Pull request merged**